holdings_maintenance/pds4/pds4infoshelf.py

```
@@ -0,0 +1,926 @@
+# Syntax:
+#   pds4infoshelf.py --task path [path ...]
+#
+# Enter the --help option to see more information.
+################################################################################
+
+import argparse
+import datetime
+import glob
+import os
+import re
+from pathlib import Path
```

Flake8 (7.3.0)

[error] 'pathlib.Path' imported but unused

(F401)